app/data_client.go

```diff
@@ -109,6 +109,37 @@ type TabularData struct {
 	TimeReceived  time.Time
 }
 
+// ExportTabularDataReturn represents the result of an ExportTabularData API call.
+type ExportTabularDataReturn struct {
```

Comment:
I'm assuming this was done since there are other data wrappers that call it `Return`. I personally would use the word `Response`, since it's a shadow type of the response. It's what we do for the Python SDK, as well as some other wrappers in the other app clients in the Go SDK.